Documentation-Requirements.md

```diff
@@ -79,3 +80,21 @@ utility.
       ```
 
   ```
+
+# Images
+
+All binary images files must be in a standard and well supported format such
```

Comment:
Lines 86-90 suggested rewrite: All binary image files must be in a standard and well-supported format such as PNG. This format is preferred for vector graphics such as diagrams because the information is stored more efficiently, leading to smaller file sizes. JPEG images are acceptable, but this format is more appropriate to store photographic images.